Document/0x05f-Testing-Local-Authentication.md

```diff
@@ -200,7 +247,7 @@ public void authenticationSucceeded(FingerprintManager.AuthenticationResult resu
 }
 ```
 
-##### Fingerprint Authentication using an Asymmetric Key Pair
+The following section describes how to do fingerprint authentication using an asymmetric key pair.

```

Comment:
It's not a section anymore, right? I suggest this change or removing the line completely.

```suggestion
The following describes how to do fingerprint authentication using an asymmetric key pair.
```